package.json

```diff
@@ -28,7 +32,8 @@
     "@hapi/wreck": "18.0.1",
     "@hapi/yar": "11.0.1",
     "applicationinsights": "2.3.5",
-    "ffc-auth": "0.1.0",
+    "ffc-auth": "0.12.0",
+    "ffc-messaging": "^2.9.0",
```

Comment:
No need for the ^

You can update your npm profile so it doesn't add them by default.  Makes it easier not to add them by mistake.  No idea why it's the default...

Reply:
Noted, have removed it manually but will set up my NPM profile.